mllib/src/main/scala/org/apache/spark/ml/feature/Bucketizer.scala

```diff
@@ -96,9 +99,71 @@ final class Bucketizer @Since("1.4.0") (@Since("1.4.0") override val uid: String
   def setHandleInvalid(value: String): this.type = set(handleInvalid, value)
```

Comment:
We should make it clear that in the multi column case, the invalid handling is applied to all columns (so for `error` it will throw the error if any invalids are found in any column, for `skip` it will skip rows with any invalids in any column, etc)